lib/testing/zone-testing.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+// load test related files into bundle in correct order
+import '../zone-spec/long-stack-trace';
+import '../zone-spec/proxy';
+import '../zone-spec/sync-test';
+import '../jasmine/jasmine';
```

Comment:
this makes an assumption about zone and jasmine being installed as siblings. Why not just `import jasmine`?

Reply:
Sorry I am not sure I understand your suggestion, I just import `../jasmine/jasmine` which include `zone patch` for jasmine method.

Reply:
Oh, I see, I thought this was importing jasmine library itself, but it's https://github.com/angular/zone.js/blob/master/lib/jasmine/jasmine.ts

The user does need to ensure that jasmine itself is loaded before the zone-testing bundle

Reply:
yes, if `jasmine` is not loaded, here will throw error `Missing: jasmine.js`